### PR TITLE
Handle no result.

### PR DESCRIPTION
**Pull request by purban**

### CODE REVIEW

File: app/scripts/main.js [hunk 1]
```diff
@@ -6,18 +6,19 @@ $(document).ready(function() {
   // ==============
 
   // Replace with your own values
-  var APPLICATION_ID = 'latency';
-  var SEARCH_ONLY_API_KEY = '6be0576ff61c053d5f9a3225e2a90f76';
-  var INDEX_NAME = 'bestbuy';
-  var HITS_PER_PAGE = 10;
+  var APPLICATION_ID       = 'latency';
```
Comment (vvo) [inline app/scripts/main.js]: If you want to switch the style to align spaces, do it in another PR, it makes the diff unreadable.

I do not recommend indenting spaces for that same reason (unreadable diffs when adding a new longer variable).

We stopped doing it in the JS projects.

Can you remove it?

Comment (purban) [inline app/scripts/main.js]: Yup, coming right up!

**Comment by purban** [comment]
Ok to merge? @vvo @redox

File: app/index.html [hunk 2]
```diff
@@ -105,6 +104,22 @@ <h2 class="hit-name">{{{ _highlightResult.name.value }}}</h2>
     </div>
   </script>
 
+  <!-- No-Results template -->
+  <script type="text/template" id="no-results-template">
+    <div id="no-results-message">
+    <p>We didn&#39;t find any results for the search <em>"{{ query }}"</em>.</p>
```
Comment (vvo) [inline app/index.html]: No need for `&#39;`, `'` works everywhere

Comment (vvo) [inline app/index.html]: gonna change it for you then we are good

File: app/images/remove.svg [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (vvo) [inline app/images/remove.svg]: why SVG since all current images are in PNG?

Comment (vvo) [inline app/images/remove.svg]: There is an icons.sketch maybe use it, add your icon and export a png? Wonder who did this .sketch, I do not have sketch :)

File: app/index.html (hunk 2 above)
Comment (purban) [inline app/index.html]: It was for syntax highlighting, but I agree that it is more readable with a ' rather than a &#39;. :)

File: app/images/remove.svg (hunk 3 above)
Comment (vvo) [inline app/images/remove.svg]: @purban
